bokeh/io/export.py

```diff
@@ -222,15 +227,12 @@ def save_layout_html(obj, resources=INLINE, **kwargs):
             obj.plot_height = kwargs.get('height', old_height)
             obj.plot_width = kwargs.get('width', old_width)
 
-    html_path = NamedTemporaryFile(suffix=".html").name
     save(obj, filename=html_path, resources=resources, title="")
 
```

Comment:
I think I'd like this better if `save` could just accept the named tmpfile object directly, but not worth changing here.